docs/testing-with-django.md

```diff
@@ -242,7 +242,10 @@ typically simulate their behavior using mocks.
 modules.  You can use the `--coverage` option to generate coverage
 reports, and new code should have 100% coverage, which generally requires
 testing not only the "happy path" but also error handling code and
-edge cases.  Note that `test-backend --coverage` will assert that
+edge cases.  It will generate a nice HTML report that you can view in
+a browser that can access your development enviroment.
```

Comment:
this sounds slightly awkward to me...maybe say "can access your development server"